packages/sanity/src/_internal/cli/actions/schema/storeSchemasAction.ts

```diff
@@ -0,0 +1,138 @@
+import {readFileSync} from 'node:fs'
+import path, {join, resolve} from 'node:path'
+
+import {type CliCommandArguments, type CliCommandContext} from '@sanity/cli'
+import chalk from 'chalk'
+
+import {
+  type CreateManifest,
+  type ManifestSchemaType,
+  type ManifestWorkspaceFile,
+} from '../../../manifest/manifestTypes'
+
+export interface StoreManifestSchemasFlags {
+  'path'?: string
+  'workspace'?: string
+  'id-prefix'?: string
+  'schema-required'?: boolean
+  'verbose'?: boolean
+}
+
+export default async function storeManifestSchemas(
+  args: CliCommandArguments<StoreManifestSchemasFlags>,
+  context: CliCommandContext,
+): Promise<Error | undefined> {
+  const flags = args.extOptions
+  const workspaceName = flags.workspace
+  const idPrefix = flags['id-prefix']
+  const verbose = flags.verbose
+  const {output, workDir, apiClient} = context
+
+  const defaultOutputDir = resolve(join(workDir, 'dist'))
+
+  const outputDir = resolve(defaultOutputDir)
+  const defaultStaticPath = join(outputDir, 'static')
+
+  const staticPath = flags.path ?? defaultStaticPath
+
+  const spinner = output.spinner({}).start('Storing schemas')
+
+  try {
+    const manifestPath = path.resolve(process.cwd(), staticPath)
+    const client = apiClient({
+      requireUser: true,
+      requireProject: true,
+    }).withConfig({apiVersion: 'v2024-08-01'})
+
+    const projectId = client.config().projectId
+
+    let manifest: CreateManifest
+
+    try {
+      manifest = JSON.parse(readFileSync(`${manifestPath}/create-manifest.json`, 'utf-8'))
```

Comment:
There is a `MANIFEST_FILENAME` in the codebase, you should move that to a constants file and reuse it everywhere.